deploy/exec/av-controller/config/set-param.ts

```diff
@@ -0,0 +1,70 @@
+import { ConfigFileHelper } from "./../../../helper/config-file-helper";
+import { AutomatedVaultController__factory } from "../../../../typechain/factories/AutomatedVaultController__factory";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { getDeployer, isFork } from "../../../../utils/deployer-helper";
+import { getConfig } from "../../../entities/config";
+import { compare } from "../../../../utils/address";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  /*
+  ░██╗░░░░░░░██╗░█████╗░██████╗░███╗░░██╗██╗███╗░░██╗░██████╗░
+  ░██║░░██╗░░██║██╔══██╗██╔══██╗████╗░██║██║████╗░██║██╔════╝░
+  ░╚██╗████╗██╔╝███████║██████╔╝██╔██╗██║██║██╔██╗██║██║░░██╗░
+  ░░████╔═████║░██╔══██║██╔══██╗██║╚████║██║██║╚████║██║░░╚██╗
+  ░░╚██╔╝░╚██╔╝░██║░░██║██║░░██║██║░╚███║██║██║░╚███║╚██████╔╝
+  ░░░╚═╝░░░╚═╝░░╚═╝░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝╚═╝░░╚══╝░╚═════╝░
+  Check all variables below before execute the deployment script
+  */
+
+  const CREDITOR_ADDRS: string[] = ["0xBCbA5F83f450a9e358964B4e306339d1F64B630e"];
+  const PRIVATE_VAULT_ADDRS: string[] = ["0xC57876a95A4f31a0A4FDB0329Fc78e00B092cC94"];
```

Comment:
improve
```suggestion
  const CREDITOR_ADDRS: string[] = ["0xBCbA5F83f450a9e358964B4e306339d1F64B630e"];
  const PRIVATE_VAULT_ADDRS: string[] = ["0xC57876a95A4f31a0A4FDB0329Fc78e00B092cC94"];
```
improve using creditor name and private-vault-symbol